pkg/rancher-desktop/typings/electron-ipc.d.ts

```diff
@@ -78,6 +79,8 @@ export interface IpcMainEvents {
   'preferences/load': () => void;
 
   'help/preferences/open-url': () => void;
+
+  'extensions-list': () => void;
```

Comment:
Are you okay with calling this `extensions/list` instead? (Also for `IpcRendererEvents`, below). We have a mix of slash-separated things and dash-separated things, but it looks like the newer ones use slash.